### PR TITLE
point to bucket

### PR DESCRIPTION
**Pull request by stdavis**
Related to https://github.com/agrc/Crash-db/pull/8

### CODE REVIEW

File: GruntFile.js [hunk 1]
```diff
@@ -251,26 +241,6 @@ module.exports = function (grunt) {
                 }
             }
         },
-        shell: {
-            options: {
-                stdout: true
-            },
-            bootstrapMapServices: {
-                command: 'python scripts/publish_mxd.py'
```
Comment (steveoh) [inline GruntFile.js:259]: **question**: this still seems useful. I'm curious about your decision to remove it.

File: GruntFile.js [hunk 2]
```diff
@@ -94,16 +94,6 @@ module.exports = function (grunt) {
         connect: {
             uses_defaults: {}
         },
-        copy: {
```
Comment (steveoh) [inline GruntFile.js]: **thought**: I think a part of this is required for the website to have the change log.

File: src/app/config.js [hunk 3]
```diff
@@ -15,7 +15,39 @@ define([
...
+
+    if (has('agrc-build') === 'prod') {
+        // crashmapping.utah.gov
+        apiKey = 'AGRC-74CDA9DA213937';
+        quadWord = 'nixon-flex-modest-stamp';
+        bucketName = 'ut-dts-agrc-crash-prod-data';
+    } else if (has('agrc-build') === 'stage') {
+        // test.mapserv.utah.gov
+        apiKey = 'AGRC-FFCDAD6B933051';
+        quadWord = 'opera-event-little-pinball';
```
Comment (steveoh) [inline src/app/config.js]:
```suggestion
        // crashmapping.dev.utah.gov
        apiKey = 'AGRC-FE1B257E901672';
        quadWord = 'wedding-tactic-enrico-yes';
```

**Review by steveoh — COMMENTED** [review]
this is a good start. I'd like to understand some of your decisions and then move forward with this. Thanks!

File: GruntFile.js (hunk 1 above)
Comment (stdavis) [reply GruntFile.js:259]: I couldn't find the `publish_mxd.py` file anywhere in the project. I assumed that this was just clean up of unused code.

Comment (steveoh) [reply GruntFile.js:259]: It looks like it was deleted in this pr. https://github.com/agrc/Crash-web/blob/77abcd559638a41fe6b050823ebec17028f21ac5/scripts/publish_mxd.py

File: GruntFile.js (hunk 2 above)
Comment (stdavis) [reply GruntFile.js]: Yes, you're correct. I got a little trigger happy. I'll add this back in...

File: GruntFile.js (hunk 1 above)
Comment (stdavis) [reply GruntFile.js:259]: Do you want me to bring it back? Looks like py2.

Comment (steveoh) [reply GruntFile.js:259]: 😳 i think so to have something to update to py3

**Pull request merged**